CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 - **UIViewController**
   - Added `addChildViewController(_:toContainerView)` to easily add child view controllers. Accepts a `UIViewController` and a `UIView` to add the child's view to. 
   - Added `removeViewAndControllerFromParentViewController()` to remove a `UIViewController` from its parent.
+- **UIEdgeInsets**
+  - Added  `insetBy(top:left:bottom:right:)` and `insetBy(horizontal:vertical)` to creates an `UIEdgeInsets` based on current value and  adjusted by given offsets. [#500](https://github.com/SwifterSwift/SwifterSwift/pull/532) by [VincentSit](https://github.com/VincentSit).
```

Comment:
The PR number is describe as 500? I believe it's 532 :))

Reply:
Thank you, fixed.